openjdk/mmtkHeap.cpp

```diff
@@ -179,7 +179,7 @@ bool MMTkHeap::is_in_reserved(const void* p) const {
     //guarantee(false, "is in not supported");
 
     //printf("calling MMTkHeap::is_in_reserved\n");
-    return p >= (void*) _start && p < (void*) _end;
+    return p >= (void*) _start && p < (void*) _end || is_in_reserved_malloc(p) == 1;
```

Comment:
Same here

Reply:
Looks like the `is_in_reserved` and `is_in` are duplications. You can just indirect one to the other to reduce the duplication, instead of copying the code twice.

Reply:
These two functions are present in the master branch as well.
They are not added by Paige.
Anyway, I'll redirect one to the other for now.